packages/devtools_app/integration_test/test_infra/test_app_driver.dart

```diff
@@ -106,34 +109,78 @@ abstract class _TestApp with IOMixin {
     _vmServiceWsUri =
         convertToWebSocketUrl(serviceProtocolUrl: _vmServiceWsUri);
 
-    await started;
+    // Now await the started event; if it had already happened the future will
+    // have already completed.
+    final startedResult = await started;
+    _currentRunningAppId = (startedResult[FlutterDaemonConstants.params.key]!
```

Comment:
Consider creating variables for the `values` being casted to make it easier to read